packages/x-date-pickers-pro/src/internal/hooks/useDesktopRangePicker/useDesktopRangePicker.tsx

```diff
@@ -151,7 +151,6 @@ export const useDesktopRangePicker = <
         inputProps: {
           ...externalInputProps?.inputProps,
           ...inputPropsPassedByField?.inputProps,
-          ...inputPropsPassedByPicker?.inputProps,
```

Comment:
This was previously only available because of untyped explicit `inputProps: { readOnly }` coming from `useRangePickerInputProps`. It had no other props.
That is no longer relevant.